src/webui/webui.cpp

```diff
@@ -93,7 +93,16 @@ void WebUI::init()
 #endif
 
         if (!m_httpServer->isListening()) {
-            bool success = m_httpServer->listen(QHostAddress::Any, m_port);
+			const QString addressString = pref->getWebUiAddress();
```

Comment:
Wrong indentation in these lines.

Reply:
Perhaps, the author forgot to consider project coding guidelines and use tabs instead of spaces for indentation... There are also some other coding style breaks in this PR.